weekly-standup-template.md

```diff
@@ -264,6 +264,29 @@ For example, this may be an emergent task, providing support or expertise to oth
 
 #### Other
 
+##### [Promotion, opportunities and new work venues](https://github.com/alan-turing-institute/data-safe-haven-team/issues/68)
+
+PIs update here!
+
```

Comment:
```suggestion
```

_I_ think we don't need this and we wouldn't want to discourage anyone below PI making connections.

Reply:
You are right, it was mostly to serve as guide.

I have some doubts with this story title and purpose, I don't want to duplicate so we have to think what is different from stakeholder engagement or communications. 

When we thought about this one it was in particular to know about and detect potential new venues of work, conversations with not a clear purpose yet, and more informal promotion. I do see a difference but we should make sure that is clear, for example do AI UK conversations belong here or in stakeholder engagement?

Reply:
I think they are related but distinct. Stakeholder engagement feels more like talking to your current stakeholders where as this story is more about finding new stakeholders.

I also think it would be good to encourage the PIs to contribute to all stories where they can.